extensions/gadgets/AdviceBar/AdviceBar.html

```diff
@@ -1,3 +1,5 @@
+<script src="{{cache_slug}}/extensions/gadgets/AdviceBar/AdviceBar.js"></script>
```

Comment:
Html templates for extensions/ are inlined using python. Hence, they cannot be directly interpolated using jinja which required us to add another method (`interpolate_cache_slug`) to jinja_utils.py.